### PR TITLE
Group example playbooks in "examples" dir

### PR DESCRIPTION
**Pull request by dkorn**
Notice it is now required to add `-M library/` when running the example playbooks

### CODE REVIEW

**Comment by dkorn** [comment]
@cben @simon3z @zgalor PTAL

File: examples/example_add_openshift_provider_playbook.retry [hunk 1]
```diff
@@ -0,0 +1 @@
+localhost
```
Comment (cben) [inline examples/example_add_openshift_provider_playbook.retry]: did you git add this file by mistake?

**Comment by cben** [comment]
LGTM

File: examples/example_add_openshift_provider_playbook.retry (hunk 1 above)
Comment (dkorn) [reply examples/example_add_openshift_provider_playbook.retry]: I did, removed now.
nice catch :)

**Comment by zgalor** [comment]
LGTM :+1: 
Now you can merge my PR https://github.com/dkorn/manageiq-ansible-module/pull/54 :)

**Comment by dkorn** [comment]
@zgalor I don't think I'll merge #54 for two reasons:
a. it should be part of #51, as it refers to the CM-OPS deployment
b. it should be submitted as an example to https://github.com/dkorn/hawkular-alerts-module

**Pull request merged**